docs/use/createBackups/EnablePITR.md

```diff
@@ -44,7 +44,18 @@ To enable PITR:
     4. Complete the setup wizard to schedule a full backup and enable PITR for the existing database.
 
 
-## Limitation for PostgreSQL
+## Limitations
+
+Point-In-Time Recovery (PITR) can only be used after a certain time has elapsed since the latest backup. This interval of time is referred to as the **PITR upload interval**. During this interval, PITR restore won't be available.
+
+The default **uploadInterval** values for different databases are as follows:  
+
+- PXC(Percona XtraDB Cluster) = 1 minute
+- PSMDB(Percona Server for MongoDB) = 10 minutes
+- PG(PostgreSQL) = 1 minute
```

Comment:
I think we should name these with the same names we show on the UI, i.e. MySQL and MongoDB